src/script/view_model/content/InputBarViewModel.js

```diff
@@ -328,22 +330,46 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
     }
   }
 
-  _saveDraftState(conversationEntity, text, mentions, reply) {
+  /**
+   * @param {string} storageKey - The primary key for the data
+   * @returns {Promise<any>} The loaded data
+   */
+  async _loadValue(storageKey) {
+    const data = isTemporaryClientAndNonPersistent()
+      ? await this.storageRepository.storageService.load(InputBarViewModel.CONFIG.INPUT_STORE_NAME, storageKey)
+      : loadValue(storageKey);
+    return data;
+  }
+
+  /**
+   * @param {string} storageKey - The primary key for the data
+   * @param {any} data - The data to save
+   * @returns {Promise<void>} No return value
+   */
+  async _storeValue(storageKey, data) {
+    if (isTemporaryClientAndNonPersistent()) {
```

Comment:
you could use the same pattern as the `_loadValue` function

```
return isTemporaryClient() ? await ... : storeValue()
```

Reply:
Oh wait no, this might change the signature of the function

Reply:
Yeah then it would return data where it shouldn't be.